.github/workflows/base-release.yaml

```diff
@@ -21,6 +21,9 @@ on:
         type: string
         default: false
 
+permissions:
+  contents: read
+
```

Comment:
Separated out to https://github.com/open-telemetry/opentelemetry-collector-releases/pull/1139

```suggestion
```